File: ccc.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char ** argv) {

    char * symbol;

    if (argc != 1) {
        symbol = argv[1];
    }

    if (argc == 1) {
        char * env_symbol = getenv("CCC_SYMBOL");
        if (env_symbol == NULL) {
            symbol = "";
        } else {
            symbol = env_symbol;
        }
    }

    printf("\n");
    for (int i = 0; i < 7; i++) {
        printf(" \033[%dm%s \033[m", 31 + i, symbol);
    }
    printf("\n");

    for (int i = 0; i < 7; i++) {
        printf(" \033[%dm%s \033[m", 91 + i, symbol);
    }
    printf("\n\n");
}

File: src/ccc.py
#!/usr/bin/env python3
import sys
import os

if len(sys.argv) > 1:
    symbol = sys.argv[1]
else:
    if "CCC_SYMBOL" in os.environ:
        symbol = os.getenv("CCC_SYMBOL")
    else:
        symbol = ""

sys.stdout.write("\n")

for i in range(0, 8):
    sys.stdout.write(f" \033[{30 + i}m {symbol} \033[m")

sys.stdout.write("\n")

for i in range(0, 8):
    sys.stdout.write(f" \033[{90 + i}m {symbol} \033[m")

sys.stdout.write("\n\n")

sys.stdout.flush()
